.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": ["eslint-config-cozy-app"]
```

Comment:
You can just put `"extends": ["cozy-app"]`

Reply:
Thanks, I will also update this PR https://github.com/konnectors/konitor/pull/75

Do you have the commit or the pull-request this string changed ?

Reply:
This is the way eslint works, you can ommit the `eslint-config-` before

Reply:
Thanks, I updated my PR https://github.com/konnectors/konitor/pull/75/commits/77c5c6a1b6176071f3d69f29a1ec714141c09d02